rasa_nlu/classifiers/sklearn_intent_classifier.py

```diff
@@ -81,7 +84,7 @@ def train(self, training_data, intent_features, num_threads):
 
         # dirty str fix because sklearn is expecting str not instance of basestr...
         tuned_parameters = [{'C': [1, 2, 5, 10, 20, 100], 'kernel': [str('linear')]}]
-        cv_splits = max(2, min(5, np.min(np.bincount(y))))
+        cv_splits = max(2, min(MAX_CV_FOLDS, np.min(np.bincount(y)) / 5))  # aim for at least 5 examples in each fold
```

Comment:
X / Y is not truncated division anymore since we are using python 3 semantic, so between 2 and 5 we could end up with float numbers